link/src/main/java/com/stripe/android/link/ui/wallet/PaymentDetails.kt

```diff
@@ -0,0 +1,257 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.stripe.android.link.R
+import com.stripe.android.link.theme.MinimumTouchTargetSize
+import com.stripe.android.link.theme.linkColors
+import com.stripe.android.link.theme.linkShapes
+import com.stripe.android.model.ConsumerPaymentDetails
+import com.stripe.android.model.ConsumerPaymentDetails.Card
+import com.stripe.android.R as StripeR
+
+@Composable
+internal fun PaymentDetailsListItem(
+    paymentDetails: ConsumerPaymentDetails.PaymentDetails,
+    enabled: Boolean,
+    isSelected: Boolean,
+    isUpdating: Boolean,
+    onClick: () -> Unit,
+    onMenuButtonClick: () -> Unit
+) {
+    Column {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .defaultMinSize(minHeight = 56.dp)
+                .clickable(enabled = enabled, onClick = onClick),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            RadioButton(
+                selected = isSelected,
+                onClick = null,
+                modifier = Modifier.padding(start = 20.dp, end = 6.dp),
+                colors = RadioButtonDefaults.colors(
+                    selectedColor = MaterialTheme.linkColors.actionLabelLight,
+                    unselectedColor = MaterialTheme.linkColors.disabledText
+                )
+            )
+            Column(
+                modifier = Modifier
+                    .padding(vertical = 8.dp)
+                    .weight(1f)
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    PaymentDetails(paymentDetails = paymentDetails)
+
+                    if (paymentDetails.isDefault) {
+                        DefaultTag()
+                    }
+
+                    val showWarning = (paymentDetails as? Card)?.isExpired ?: false
+                    if (showWarning && !isSelected) {
+                        Icon(
+                            painter = painterResource(R.drawable.stripe_link_error),
+                            contentDescription = null,
+                            modifier = Modifier.size(20.dp),
+                            tint = MaterialTheme.linkColors.errorText
+                        )
+                    }
+                }
+            }
+
+            MenuAndLoader(
+                enabled = enabled,
+                isUpdating = isUpdating,
+                onMenuButtonClick = onMenuButtonClick
+            )
+        }
+        TabRowDefaults.Divider(
```

Comment:
should the divider be included in the list itself, rather than on each individual list item? This seems like it would give us less control over something like not having a divider shown at the bottom of the list